integrations/server/test_covidcast.py

```diff
@@ -79,6 +79,41 @@ def test_round_trip(self):
       'message': 'success',
     })
 
+  def test_uri_too_long(self):
+    """Make a simple round-trip with some sample data."""
```

Comment:
small thing, but do you mind updating this description to say we're testing get/post behavior?

Reply:
oops this was copy pasted from another test, I've updated the docstrings.